### PR TITLE
Suppress error display by deprecating style methods.

### PR DESCRIPTION
**Pull request by Noma-Machiko**
# Before correction
self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery").style(grid=image_columns)

# After correction
self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery", grid=image_columns)

...etc

### CODE REVIEW

**Review by Noma-Machiko — COMMENTED** [review]
tested

**Comment by noisefloordev** [comment]
I just made the same change before realizing there was already a PR, this fixes noisy warning spam on startup with the version of Gradio the webui is using in 1.6:

```
sd-webui\extensions\stable-diffusion-webui-dataset-tag-editor\scripts\main.py:218: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  with gr.Row().style(equal_height=False):
sd-webui\extensions\stable-diffusion-webui-dataset-tag-editor\scripts\tag_editor_ui\block_dataset_gallery.py:25: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery").style(grid=image_columns)
sd-webui\extensions\stable-diffusion-webui-dataset-tag-editor\scripts\tag_editor_ui\block_dataset_gallery.py:25: GradioDeprecationWarning: The 'grid' parameter will be deprecated. Please use 'columns' in the constructor instead.
  self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery").style(grid=image_columns)
sd-webui\extensions\stable-diffusion-webui-dataset-tag-editor\scripts\tag_editor_ui\tab_filter_by_selection.py:35: GradioDeprecationWarning: The `style` method is deprecated. Please set these arguments in the constructor instead.
  self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery").style(grid=image_columns)
sd-webui\extensions\stable-diffusion-webui-dataset-tag-editor\scripts\tag_editor_ui\tab_filter_by_selection.py:35: GradioDeprecationWarning: The 'grid' parameter will be deprecated. Please use 'columns' in the constructor instead.
  self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery").style(grid=image_columns)
```

File: scripts/tag_editor_ui/block_dataset_gallery.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def create_ui(self, image_columns):
             self.btn_hidden_set_index = gr.Button(elem_id="dataset_tag_editor_btn_hidden_set_index")
             self.nb_hidden_image_index = gr.Number(value=None, label='hidden_idx_next')
             self.nb_hidden_image_index_prev = gr.Number(value=None, label='hidden_idx_prev')
-        self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery").style(grid=image_columns)
+        self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery", grid=image_columns)
```
Comment (Sj-Si) [inline scripts/tag_editor_ui/block_dataset_gallery.py:25]: Incorrect argument to `gr.Gallery`. Should be `columns=image_columns`, not `grid=image_columns`:
```diff
-self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery", grid=image_columns)
+self.gl_dataset_images = gr.Gallery(label='Dataset Images', elem_id="dataset_tag_editor_dataset_gallery", columns=image_columns)
```

File: scripts/tag_editor_ui/tab_filter_by_selection.py [hunk 2]
```diff
@@ -32,7 +32,7 @@ def create_ui(self, image_columns:int):
                 self.btn_add_image_selection = gr.Button(value='Add selection [Enter]', elem_id='dataset_tag_editor_btn_add_image_selection')    
                 self.btn_add_all_displayed_image_selection = gr.Button(value='Add ALL Displayed')    
 
-            self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery").style(grid=image_columns)
+            self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery", grid=image_columns)
```
Comment (Sj-Si) [inline scripts/tag_editor_ui/tab_filter_by_selection.py:35]: Incorrect argument to `gr.Gallery`. Should be `columns=image_columns`, not `grid=image_columns`:
```diff
-self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery", grid=image_columns)
+self.gl_filter_images = gr.Gallery(label='Filter Images', elem_id="dataset_tag_editor_filter_gallery", columns=image_columns)
```

**Review by Sj-Si — COMMENTED** [review]
Issue with some of the `gr.Gallery` arguments.
I'm using Gradio v3.41.2.

**Comment by toshiaki1729** [comment]
maybe fixed in #93

**Pull request closed without merging**